app/models/record.server.ts

```diff
@@ -0,0 +1,98 @@
+import type { Record, User } from '@prisma/client';
```

Comment:
I wonder if this file is achieving anything.  Maybe we'd be better to use Prisma directly in your other file, since all you're doing here is calling the Prisma functions with the passed arguments (i.e., you're not adding a lot of value that we couldn't do directly in other code).

Reply:
It's true that I started with direct `prisma` call. If we do that way, I think we would barely need any models. I like removing my model since I could reduce the code. Will remove it.